src/CMakeLists.txt

```diff
@@ -171,11 +171,6 @@ set(INCLUDES
 )
 set(INCLUDES ${INCLUDES} PARENT_SCOPE)
 
-set(SYSTEMINCLUDES
-  ${CMAKE_BINARY_DIR}/messages
-)
-set(SYSTEMINCLUDES ${SYSTEMINCLUDES} PARENT_SCOPE)
```

Comment:
:+1: might have been from when he used C nanopb in the past. Completely switched to Rust/prost in the meantime.